iis-common/src/main/java/eu/dnetlib/iis/common/counter/PigCounters.java

```diff
@@ -0,0 +1,145 @@
+package eu.dnetlib.iis.common.counter;
+
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+
+/**
+ * Class that simplifies use of pig counters.
+ * 
+ * @author madryk
+ */
+public class PigCounters {
+
+    private Map<String, JobCounters> jobCountersMap;
+    
+    
+    //------------------------ CONSTRUCTORS --------------------------
+    
+    /**
+     * Default constructor
+     * 
+     * @param jobCountersList - counters of map-reduce jobs run inside pig job
+     */
+    public PigCounters(List<JobCounters> jobCountersList) {
+        Preconditions.checkNotNull(jobCountersList);
+        Preconditions.checkArgument(jobCountersList.size() > 0);
+        
+        this.jobCountersMap = Maps.newHashMap();
+        
+        for(JobCounters jobCounters : jobCountersList) {
+            this.jobCountersMap.put(jobCounters.getJobId(), jobCounters);
+        }
+    }
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Returns list of job ids
+     */
+    public List<String> getJobIds() {
+        return Lists.newArrayList(jobCountersMap.keySet());
+    }
+    
+    /**
+     * Returns {@link JobCounters} with provided job id
+     * or <code>null</code> if job does not exists.
+     */
+    public JobCounters getJobCounters(String jobId) {
+        return jobCountersMap.get(jobId);
+    }
+    
+    /**
+     * Returns job id with provided alias
+     * or <code>null</code> if job does not exists.
+     */
+    public String getJobIdByAlias(String jobAlias) {
```

Comment:
This method is a bit strange. It returns the jobId of the <b>first</b> (no information given in the comment) encountered JobCounters which contains the given alias.